app/scripts/modules/core/src/reactShims/ngReact.ts

```diff
@@ -54,7 +54,7 @@ export class NgReactInjector extends ReactInject {
   public StageSummaryWrapper: React.ComponentClass<IStageSummaryWrapperProps>                   = angular2react('stageSummary', stageSummaryComponent, this.$injectorProxy) as any;
   public StepExecutionDetailsWrapper: React.ComponentClass<IStepExecutionDetailsWrapperProps>   = angular2react('stepExecutionDetails', stepExecutionDetailsComponent, this.$injectorProxy) as any;
   public TargetSelect: React.ComponentClass<ITargetSelectProps>                                 = angular2react('targetSelect', targetSelectComponent, this.$injectorProxy) as any;
-  public TaskMonitorWrapper: React.ComponentClass<ITaskMonitorProps>                            = angular2react('taskMonitorWrapper', taskMonitorWrapperComponent, this.$injectorProxy) as any;
+  public TaskMonitorWrapper: React.FunctionComponent<ITaskMonitorProps>                         = TaskMonitorWrapper;
```

Comment:
can we just completely remove this from ngReact and import from `core/task/monitor/TaskMonitorWrapper` in the other code?

Reply:
Yeah, for sure, but I would like to do that in a separate PR, since it will touch another 30 or so files. I was thinking I'd do that the same time I de-angularize the confirmation modal service, since that is also a breaking change in our internal build.